shared/src/commonMain/kotlin/dev/johnoreilly/confetti/Model.kt

```diff
@@ -6,8 +6,10 @@ import kotlinx.datetime.TimeZone
 import kotlinx.datetime.toInstant
 
 fun SessionDetails.isBreak() = this.type == "break"
+fun SessionDetails.isService() = this.type == "service"
```

Comment:
Will add an enum for that so we have better type safety and we can map various conferences to that enum